FordLabs/retroquest
# Adds character countdown to column titles!

## Overview
This fixes issue #4 by adding a character countdown to the thought column titles

### Demo
<img width="372" alt="screen shot 2018-07-13 at 2 24 10 pm" src="https://user-images.githubusercontent.com/6293337/42707512-6deebc82-86a8-11e8-9d24-806b3411143f.png">

## Testing Instructions
Log into the either the styleguide page or a team and edit a column title. You should see a character countdown appear below it, in the far right corner, just like the thought / action textfield.
